examples/u2f/webpack.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
```

Comment:
Should this file be in the configs with the other webpack config?

Reply:
This is build config for example.

Reply:
The convention I've seen for multiple webpack configs is to have them all together and then just name the file according to purpose (e.g. `webpack.dev.config`, `webpack.prod.config`, and then we could do `webpack.test.config` too). I guess it's not a big deal, but might be nice to have them all in one place when someone is going through and trying to figure out how to build something similar themselves. But it also wouldn't be hard to find it next to the example either :)

Reply:
The project build webpack configurations are in https://github.com/bcoin-org/bledger/tree/device-u2f/.config folder. This is specific to the example itself. Also can be used for boilerplate when building something using this library.